guide/14-deep-learning/faster-rcnn-object-detector.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>Replace title to read the title of the page. We need this so the browser tab, SE indexers get them correctly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added a title to the guide with the same name as the notebook file. Does this help?

Reply:
<p>yes, thanks</p>